suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -951,3 +951,26 @@ export const getTxHeaderSymbol = (transaction: WalletAccountTransaction) => {
 
     return symbol;
 };
+
+export const groupTransactionsBySymbol = (txs: WalletAccountTransaction[]) => {
+    const groupedTxs: Record<string, WalletAccountTransaction[]> = {};
```

Comment:
Please use branded types instead of just string, same as `FiatRateKey` for example